### PR TITLE
added missing docs, fixed warnings and 80c

### PR DESCRIPTION
**Pull request by caarlos0**
- added missing docs
- fixed all govet warnings
- tried to make all lines < 80c

### CODE REVIEW

**Comment by caarlos0** [comment]
(seems like the ci is not running on the pr)

File: cleanup.go [hunk 1]
```diff
@@ -11,7 +11,9 @@ var defaultInterval = time.Minute * 5
 // sessions from the database.
 //
 // The design is based on https://github.com/yosssi/boltstore
-func (db *PGStore) Cleanup(interval time.Duration) (chan<- struct{}, <-chan struct{}) {
+func (db *PGStore) Cleanup(
```
Comment (elithrar) [inline cleanup.go]: It's not worth splitting function declarations across lines for the sake of 80 cols. The standard lib doesn't do this.

File: pgstore.go [hunk 2]
```diff
@@ -22,16 +23,17 @@ type PGStore struct {
 
 // Session type
 type Session struct {
-	Id         int64     `db: "id"`
-	Key        string    `db: "key"`
-	Data       string    `db: "data"`
-	CreatedOn  time.Time `db: "created_on"`
-	ModifiedOn time.Time `db: "modified_on"`
-	ExpiresOn  time.Time `db: "expires_on"`
+	ID         int64     `db:"id"`
```
Comment (elithrar) [inline pgstore.go]: This is an API breaking change (however minor it may be)

File: pgstore.go [hunk 3]
```diff
@@ -13,6 +13,7 @@ import (
 	_ "github.com/lib/pq"
 )
 
+// PGStore main type
```
Comment (elithrar) [inline pgstore.go]: Should say "PGStore represents the currently configured session store" or something along those lines.

File: cleanup.go (hunk 1 above)
Comment (caarlos0) [inline cleanup.go]: ok, changing method declarations back

File: pgstore.go (hunk 3 above)
Comment (caarlos0) [inline pgstore.go]: indeed.

**Comment by antonlindstrom** [comment]
@caarlos0 Thanks for the PR! The API breaking change is something that I'm not very comfortable doing at this point.

I understand why you'd like to break lines longer than 80 characters, but I'm not a big fan of it in this case. I guess there were one or two lines that are a bit too long that should be wrapped but as there is no standard on line length limit in Go I think it's alright to stay this way.

The `go vet` and the documentation is awesome! I'm happy to merge them.

Thanks!

**Comment by caarlos0** [comment]
@antonlindstrom ok, all fixed

**Comment by antonlindstrom** [comment]
@caarlos0 Awesome, thanks! Do you mind squashing the commits into one commit?

**Comment by caarlos0** [comment]
@antonlindstrom done

**Comment by antonlindstrom** [comment]
@caarlos0 Awesome, thanks! :bow:

**Comment by caarlos0** [comment]
@antonlindstrom thanks :)

**Pull request merged**